diffkemp/simpll/Config.cpp

```diff
@@ -161,9 +169,17 @@ Config::Config(std::string FirstFunName,
     }
 
     std::vector<std::string> debugTypes;
-    if (Verbose) {
-        // Enable debugging output in passes
-        debugTypes.emplace_back(DEBUG_SIMPLL);
+    if (VerbosityOpt > 0) {
```

Comment:
Should be like this, I believe:
```suggestion
    if (Verbosity > 0) {
```
`VerbosityOpt` is not relevant here as this constructor is used when SimpLL is **not** invoked from CLI.